cnd/src/http_api/mod.rs

```diff
@@ -38,10 +38,19 @@ use serde::{
     ser::SerializeStruct,
     Deserialize, Deserializer, Serialize, Serializer,
 };
+use std::ops::Deref;
 
 #[derive(Debug)]
 pub struct Http<I>(pub I);
 
+impl<I> Deref for Http<I> {
```

Comment:
:question: Where is this used? Please explain :smiley:

Reply:
I think it is no longer used, can removed it :)